molecule/testinfra/common/test_system_hardening.py

```diff
@@ -7,34 +7,6 @@
 testinfra_hosts = [sdvars.app_hostname, sdvars.monitor_hostname]
 
 
-@pytest.mark.parametrize(
```

Comment:
IMO keeping the testinfra checks makes sense, in case the metapackage-set values get borked somehow.